org.freedesktop.Sdk.Extension.wxwidgets.yaml

```diff
@@ -0,0 +1,209 @@
+id: org.freedesktop.Sdk.Extension.wxwidgets
```

Comment:
I would rather take the BaseApp route instead.

Reply:
As written in the 1st post, this is designed for wxWidgets development with Codeblocks IDE or any other IDEs.  
Building WebKitGTK without ccache hits takes a substantial amount of time, and I would not bother with it if I didn't want an SDK extension.  
Even if there was a base application, I would still submit this extension.

Reply:
ah ok, nevermind then.